components/dashboard/tailwind.config.js

```diff
@@ -65,6 +65,11 @@ module.exports = {
             'grayscale': 'grayscale(1)',
         },
     },
+    variants: {
+        extend: {
+            opacity: ['disabled'],
```

Comment:
**thought**: Ideally we should not use opacity for disabled states but different colors, however, let's leave this is out of the scope of this milestone. FWIW, our design specs have not reached this level of maturity yet, it's still missing aspects like tabindex, active, focus state, etc for many elements. Food for thought! 🍔